ngraph/test/frontend/shared/src/main.cpp

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <iostream>
+
+#include <ngraph/file_util.hpp>
+#include "gtest/gtest.h"
+
+#ifdef __APPLE__
+#include <mach-o/dyld.h>
+#endif
+#ifdef _WIN32
+#include <Windows.h>
+#else
+#include <limits.h>
+#endif
+
+using namespace std;
+
+namespace FrontEndTestUtils
+{
+    int run_tests(int argc, char** argv)
+    {
+        ::testing::InitGoogleTest(&argc, argv);
+        int rc = RUN_ALL_TESTS();
+        return rc;
+    }
+
+    std::string get_current_executable_path()
```

Comment:
can we use https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/ie_test_utils/common_test_utils/file_utils.hpp#L243 from `commonTestUtils`? I see it's linked to shared FE tests